sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -1008,13 +1008,9 @@ object DDLUtils extends Logging {
     if (!catalog.isTempView(tableMetadata.identifier)) {
       tableMetadata.tableType match {
         case CatalogTableType.VIEW if !isView =>
-          throw QueryCompilationErrors.cannotAlterViewWithAlterTableError(
-            viewName = tableMetadata.identifier.table
-          )
+          throw QueryCompilationErrors.cannotAlterViewWithAlterTableError()
```

Comment:
Why do I see this change in this CR? This change is already present in the latest code.